{{cookiecutter.repo_name}}/tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = {py27,py35,py36}-django{18,111},{py35,py36}-django20
+envlist = {py35,py36,py37,py38}-django{111,20,21,22}
```

Comment:
Do we really want to include Django 2.0 and 2.1?  We won't be using those versions, and it doubles the test matrix/